packager/app/test/packager_test.py

```diff
@@ -593,6 +593,16 @@ def testPackageWithLiveProfileAndKeyRotationAndNonDashIfIop(self):
         self.mpd_output,
         'bear-640x360-av-live-cenc-rotation-non-iop-golden.mpd')
 
+  def testPackageWithEncryptionOfOnlyVideoStream(self):
+    self.packager.Package(
+        self._GetStreams(['audio', 'video'],
```

Comment:
Prefer something like 

self.packager.Package(self._GetStreams(['audio,skip_encryption=1','video']), ....)

It is more flexible and probably clearer as well.

The changes in _GetStreams is not needed.